### PR TITLE
feat: iterate on configurator

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- General reiterations configurator

https://github.com/cowprotocol/cowswap/assets/31534717/0c10c06f-c7a3-4d72-910c-a94eb7615fc5

### CODE REVIEW

File: apps/widget-configurator/src/app/configurator/index.tsx [hunk 1]
```diff
@@ -43,6 +45,11 @@ const DEFAULT_STATE = {
 
 const UTM_PARAMS = 'utm_content=cow-widget-configurator&utm_medium=web&utm_source=widget.cow.fi'
 
+const LINKS = [
+  { icon: <LanguageIcon />, label: 'Widget web', url: `https://cow.fi/widget/?${UTM_PARAMS}` },
+  { icon: <ChromeReaderModeIcon />, label: 'Developer docs', url: `https://docs.cow.fi/?${UTM_PARAMS}` },
+]
```
Comment (anxolin) [inline apps/widget-configurator/src/app/configurator/index.tsx:51]: Should we move the URLs to a const?

**Review by anxolin — APPROVED** [review]
Looks great!

**Review by shoom3301 — APPROVED** [review]

**Comment by fairlighteth** [comment]
Consolidated waterfall PR, with the latest being https://github.com/cowprotocol/cowswap/pull/3366#issuecomment-1803933799

**Review by anxolin — COMMENTED** [review]
only comment is if we should delete `fontsource/inter` if we don't use it

**Pull request merged**